environment.yaml

```diff
@@ -12,3 +12,8 @@ dependencies:
   - openssl
   - ripgrep
   - poetry=1.1
+  - mykrobe
```

Comment:
once version 0.11 is out we will need to pin this

Reply:
For now we have `0.10` fixed: https://github.com/mbhall88/tbpore/pull/8/commits/b6a5797af99166b8194d3d790ea8ee17ad0af3d4